packages/cosmwasm-stargate/src/queries/wasm.spec.ts

```diff
@@ -14,6 +14,11 @@ import {
 } from "@cosmjs/stargate";
 import { assert, assertDefined } from "@cosmjs/utils";
 import Long from "long";
+import {
+  MsgExecuteContractEncodeObject,
+  MsgInstantiateContractEncodeObject,
+  MsgStoreCodeEncodeObject,
+} from "src/encodeobjects";
```

Comment:
Shouldn't this be a relative path?

Reply:
Ugh, I have the feeling my VSCode used to use the import paths I wanted and recently switched to this.